infra/src/main/java/com/redhat/service/smartevents/infra/exceptions/mappers/ExternalUserExceptionMapper.java

```diff
@@ -25,16 +28,26 @@ public class ExternalUserExceptionMapper implements ExceptionMapper<ExternalUser
     @Override
     public Response toResponse(ExternalUserException e) {
         LOGGER.debug("Failure", e);
-        Optional<BridgeError> error = bridgeErrorService.getError(e);
+        Optional<BridgeError> error = bridgeErrorService.getError(e.getClass());
         ResponseBuilder builder = Response.status(e.getStatusCode());
         if (error.isPresent()) {
             ErrorResponse errorResponse = ErrorResponse.from(error.get());
             errorResponse.setReason(e.getMessage());
-            builder.entity(errorResponse);
+            builder.entity(ErrorsResponse.toErrors(errorResponse));
         } else {
-            LOGGER.warn("Information for exception type {} cannot be found", e.getClass());
-            builder.entity(e.getMessage());
+            builder.entity(ErrorsResponse.toErrors(unmappedException(e)));
         }
         return builder.build();
     }
+
+    private ErrorResponse unmappedException(Exception e) {
+        Optional<BridgeError> error = bridgeErrorService.getError(UnclassifiedException.class);
```

Comment:
Whether we replace this with the generic `ExternalUserException` or not, we should do this lookup in a `@PostConstruct` and simply fail to start the service if we cannot lookup the `BridgeError` we need. Take a look at `InternalPlatformExceptionMapper#init` which will cause the deployment of the Fleet Manager to exit if required `Error` is missing.